.github/workflows/deploy-to-S3.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy to S3
+on: [push]
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      REPO: ${{ github.repository }}
+      TAG: ${{ github.ref }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7.2
+    - name: Install sass
+      run: gem install sass
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 6.11.4
+    - name: script/bootstrap
+      run: script/bootstrap
+    - name: Get version
+      run: |
+            curl -H "Content-Type: application/json" \
+            -X POST \
+            -d "{ \"component\": { \"name\": \"formrenderer-base\", \"version\": \"$version\" } }" \
```

Comment:
We should embed the version here from the same source: `${{ steps.package-version.outputs.current-version}}` I don't think `$version` is available.

Reply:
We should also move this right above: `jakejarvis/s3-sync-action@master` so that we have access to the correct version number